mimic/model/heat_objects.py

```diff
@@ -135,6 +135,8 @@ def creation_response_json(self, absolutize_url):
 
 
 stack_creation = EventDescription()
+stack_check = EventDescription()
```

Comment:
There are definitely other places within mimic (the nova mock springs to mind) where non-default behaviors are used.  We want *more* usage of behaviors, generally, not less…